src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,91 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix.sensors.CANCoder;
+import com.ctre.phoenix.sensors.CANCoderConfiguration;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+
+public class ArmConstants {
+    private static final int
+            ANGLE_MOTOR_ID = 1,
+            ELEVATOR_MOTOR_ID = 2,
+            ANGLE_NEO_ID = 3,
+            ELEVATOR_NEO_ID = 4,
+            TALON_SRX_ID = 5,
+            ANGLE_ENCODER_ID = 6;
+
+    private static final CANSparkMax
+            ANGLE_MOTOR = new CANSparkMax(ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            ELEVATOR_MOTOR = new CANSparkMax(ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final CANSparkMax.IdleMode
+            ELEVATOR_IDLE_MODE = CANSparkMax.IdleMode.kCoast,
+            ANGLE_IDLE_MODE = CANSparkMax.IdleMode.kCoast;
+    private static final TalonSRX TALON_SRX = new TalonSRX(TALON_SRX_ID);
+    private static final CANCoder ANGLE_ENCODER = new CANCoder(ANGLE_ENCODER_ID);
```

Comment:
You're using the wrong cancoder import. You need to select the v6 one.

Reply:
where do I select the version? i don't see an option

Reply:
also, it's showing me an error where I made the range, idk why

Reply:
When you type CAN the auto complete will show you a few options. For each one, it'll specify its import. You just need to make sure you chose the one with v6 in its import. And the more visible difference is that the v5 one will be `CANCoder` and the v6 one will be `CANcoder`. (Same goes for configs and everything else)